src/typeahead/typeahead.directive.ts

```diff
@@ -27,7 +27,7 @@ import { TypeaheadConfig } from './typeahead.config';
 
 // eslint-disable-next-line
 type TypeaheadOption = string | Record<string | number, any>;
-type TypeaheadOptionArr = TypeaheadOption[] | Observable<TypeaheadOption[]>;
+type TypeaheadOptionArr = TypeaheadOption[] | Observable<TypeaheadOption>;
```

Comment:
I have not looked into this, but this change seems a bit odd. Is this correct?

Reply:
Based on usage on [line 485](https://github.com/valor-software/ngx-bootstrap/blob/528df9de07e6214b4cdfb4a5dfbc6737468d3fde/src/typeahead/typeahead.directive.ts#L485), that's the correct type.

Reply:
Another possibility is that they used wrong rxjs function and, instead of `from(this.typeahead)`, it should be `of(this.typeahead)` on line `485`. Then the old type could be correct.

Reply:
@heeheehi and @valorkin might have more insights into this.

Reply:
It seems like @valorkin gave my comment a thumbsup. Can we infer that they agree with my assessment then?

Reply:
Any update as far as a timeline? project in limbo due to a need to use angular 16. (comment mostly for notification)

Reply:
@valorkin @SvetlanaMuravlova